examples/up.conf.example

```diff
@@ -24,13 +24,21 @@ quiet = false
 # an invalid mode will also result in "influx". In this default mode the application
 # runs as a daemon and polls the controller at the configured interval.
 #
-# There is only one other option at this time: "influxlambda"
+# There are two other options at this time: "influxlambda" and "prometheus"
 #
 # Lambda mode makes the application exit after collecting and reporting metrics
 # to InfluxDB one time. This mode requires an external process like an AWS Lambda
 # or a simple crontab to keep the timings accurate on UniFi Poller run intervals.
+#
+# Prometheus mode opens an HTTP server on port 9130 and exports the metrics at
+# /metrics for polling collection by a prometheus server. This disables influxdb.
+# IMPORTANT: The prometheus mode is still beta and doesn't work very well.
```

Comment:
What isn't working "well"? This is a bit subjective and doesn't really help direct the user to making decisions.

Reply:
thanks! reworded.